README.markdown

```diff
@@ -87,6 +91,14 @@ For Windows:
     -L+org.eclipse.swt.win32.win32.x86.lib -L+dwt-base.lib -L/SUBSYSTEM:WINDOWS:4.0
   ```
 
+  If you use Microsoft linker:
+
+  ```
+  $ dmd main.d -I<dwt>\imp -J<dwt>\res -L/LIBPATH:<dwt>\lib ^
```

Comment:
The contents of the folder is copied to `<dwt>\res` when building.

Examples of 32-bit and 64-bit seems to right both, What had better fix either?